### ISSUE TITLE
Build fail on system with openssl 1.1.1

### ISSUE THREAD

**Issue by portaloffreedom** [mattnenterprise/rust-imap#94]
Easy fix, you just need to update the openssl library

Related Github issue:
https://github.com/sfackler/rust-openssl/issues/994

this is the error:
```$ cargo build 
   Compiling openssl v0.9.24                                                                                                                                                                                      
error: failed to run custom build command for `openssl v0.9.24`                                                                                                                                                   
process didn't exit successfully: `~/projects/rust/mail_test/target/debug/build/openssl-d6940fa3998f9474/build-script-build` (exit code: 101)
--- stderr
thread 'main' panicked at 'Unable to detect OpenSSL version', ~/.cargo/registry/src/github.com-1ecc6299db9ec823/openssl-0.9.24/build.rs:16:14
note: Run with `RUST_BACKTRACE=1` for a backtrace.
```

**Comment by jonhoo** [comment]
Are you trying to operate on the version on crates.io, or on the git version? Sadly, due to #69, crates.io hasn't seen a release in quite some time. The git version depends on `native-tls 0.2`, which doesn't suffer from that issue :)

**Comment by portaloffreedom** [comment]
Oh, I see :(

Yes, I'm running the crates.io version. I'll try the git version and I think my problem will be solved. If that's not the case, I'll write something here.

I think the issue should stay open until the crates.io version is updated. Mostly for other people that will have the same problem.

**Comment by jonhoo** [comment]
Fixed in `0.9`!
